### DISCUSSION TITLE
关于requirements中opencv的版本限制？

### DISCUSSION THREAD

**Discussion by SWHL** [PaddlePaddle/PaddleOCR#12821] [Q&A]
我很好奇是哪里限制了opencv-python的版本必须是`<=4.6.0.66`呢？

https://github.com/PaddlePaddle/PaddleOCR/blob/862b4d46edc55b8bf3d7806e7c316e11a1d5f478/pyproject.toml#L50-L51

**Comment by GreatV** [comment]
@Liyulingyue

**Reply by Liyulingyue** [reply]
没有明确的指向，建议单侧CI组织好之后测一下，或者现在可以针对常用的几个API测一下

**Reply by SWHL** [reply]
@Liyulingyue 
可以帮忙把相关API补充到现有[单元测试](https://github.com/PaddlePaddle/PaddleOCR/tree/main/tests)中吗？
